CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0091 NEW) # enable new "MSVC runtime library selection" (https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html)
 
 project(libCZI 
-      VERSION 0.62.1
+      VERSION 0.62.2
```

Comment:
Maybe bump to 0.62.3 "directly" with the prospect of [PR 115](https://github.com/ZEISS/libczi/pull/116) which proposes to increase the patch to 0.62.2 and is potentially first?

Reply:
No worries, I'll update this PR once #115 is in.

Reply:
fixed